torchtext/csrc/CMakeLists.txt

```diff
@@ -0,0 +1,93 @@
+################################################################################
+# _torchtext.so
+################################################################################
+if (BUILD_TORCHTEXT_PYTHON_EXTENSION)
+  set(LIBTORCHTEXT_COMPILE_DEFINITIONS)
+
+  # See https://github.com/pytorch/pytorch/issues/38122
+  find_library(TORCH_PYTHON_LIBRARY torch_python PATHS "${TORCH_INSTALL_PREFIX}/lib")
+  if (WIN32)
+    find_package(Python3 ${PYTHON_VERSION} EXACT COMPONENTS Development)
+    set(ADDITIONAL_ITEMS Python3::Python)
+  endif()
+  function(define_extension name sources include_dirs link_libraries definitions)
+    add_library(${name} SHARED ${sources})
+    target_compile_definitions(${name} PRIVATE "${definitions}")
+    target_include_directories(
+      ${name} PRIVATE ${Python_INCLUDE_DIR} ${include_dirs})
+    target_link_libraries(
+      ${name}
+      ${link_libraries}
+      ${TORCH_PYTHON_LIBRARY}
+      ${ADDITIONAL_ITEMS}
+      )
+    set_target_properties(${name} PROPERTIES PREFIX "")
+    if (MSVC)
+      set_target_properties(${name} PROPERTIES SUFFIX ".pyd")
+    endif(MSVC)
+    if (APPLE)
+      # https://github.com/facebookarchive/caffe2/issues/854#issuecomment-364538485
+      # https://github.com/pytorch/pytorch/commit/73f6715f4725a0723d8171d3131e09ac7abf0666
+      set_target_properties(${name} PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
+    endif()
+    install(
+      TARGETS ${name}
+      LIBRARY DESTINATION .
+      RUNTIME DESTINATION .  # For Windows
+      )
+  endfunction()
+
+  set(
+    EXTENSION_SOURCES
+    clip_tokenizer.cpp
+    clip_tokenizer.h
```

Comment:
since all the headers are now looked up from the root directory of the repo, so I think it's less confusing not to have headers listed here.

Reply:
Done!